### PR TITLE
.NET Interactive: Enable_try_dotnet_to_batch_9

### PR DESCRIPTION
**Pull request by v-mepa**
**Enable_try_dotnet_to_batch_9**

URL | Comments
-- | --
https://docs.microsoft.com/en-us/dotnet/api/system.double.tostring | Only examples 1, 3, 6, 7, 9
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.aes |  
https://docs.microsoft.com/en-us/dotnet/api/system.double.tryparse |  
https://docs.microsoft.com/en-us/dotnet/api/system.double.parse | Except for examples 3, 5. Example 7 -   let's test that one with try-dotnet-class
https://docs.microsoft.com/en-us/dotnet/api/system.double.nan |  
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.aescryptoserviceprovider
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.aesmanaged
https://docs.microsoft.com/en-us/dotnet/api/system.double.equals |  
https://docs.microsoft.com/en-us/dotnet/api/system.double.maxvalue | Only second example
https://docs.microsoft.com/en-us/dotnet/api/system.double.epsilon |  
https://docs.microsoft.com/en-us/dotnet/api/system.double.positiveinfinity |  
https://docs.microsoft.com/en-us/dotnet/api/system.double.minvalue | Only samples 2 and 4

### CODE REVIEW

**Comment by v-mepa** [comment]
FYI @mairaw 
Was little confused about the last API, hope I have updated the correct samples :), let me know.

File: xml/System/Double.xml [hunk 1]
```diff
@@ -1380,7 +1380,7 @@
   
  The result of an operation that is less than <xref:System.Double.MinValue?displayProperty=nameWithType> is <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>. In the following example, <xref:System.Double.NegativeInfinity?displayProperty=nameWithType> results from subtraction and multiplication operations when the result is less than <xref:System.Double.MinValue?displayProperty=nameWithType>.  
   
- [!code-csharp[System.Double.MinValue#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.double.minvalue/cs/minvalueex.cs#1)]
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.double.minvalue/cs/minvalueex.cs" interactive="try-dotnet" id="Snippet1":::
  [!code-vb[System.Double.MinValue#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.double.minvalue/vb/minvalueex.vb#1)]
```
Comment (mairaw) [inline xml/System/Double.xml:1384]: let's change the syntax for vb too

File: xml/System/Double.xml [hunk 2]
```diff
@@ -1389,7 +1389,7 @@
  The following code example illustrates the use of <xref:System.Double.MinValue?displayProperty=nameWithType>:  
   
  [!code-cpp[Double Example#2](~/samples/snippets/cpp/VS_Snippets_CLR/Double Example/CPP/source.cpp#2)]
```
Comment (mairaw) [inline xml/System/Double.xml:1391]: let's change the syntax for the other languages here too

File: xml/System/Double.xml [hunk 3]
```diff
@@ -1389,7 +1389,7 @@
...
- [!code-csharp[Double Example#2](~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs#2)]
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" interactive="try-dotnet-method" id="Snippet2":::
```
Comment (mairaw) [inline xml/System/Double.xml:1392]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" id="Snippet2":::
```

**Comment by v-mepa** [comment]
@mairaw I am seeing 2 commits for "update_syntax_for_other_languages" , Git asked me to pull before committing and I did and now its showing 2 commits. Should I remove 
82cc68a commit. Please advise.

**Comment by mairaw** [comment]
Let me see if I can comment on the problems introduced instead of you having to remove the commits. Because you applied my suggestions here first, you first need to do a git pull to bring the changes into your local version first.

**Comment by v-mepa** [comment]
Okay make sense now, I will keep that in mind from now on. Learning new things each day and I am loving it Maira :)

File: xml/System/Double.xml [hunk 4]
```diff
@@ -1380,18 +1380,24 @@
...
- [!code-vb[System.Double.MinValue#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.double.minvalue/vb/minvalueex.vb#1)]  
-  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.double.minvalue/cs/minvalueex.cs" interactive="try-dotnet" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.double.minvalue/vb/minvalueex.vb" id="Snippet1":::
    
   
 ## Examples  
  The following code example illustrates the use of <xref:System.Double.MinValue?displayProperty=nameWithType>:  
   
+<<<<<<< HEAD
```
Comment (mairaw) [inline xml/System/Double.xml:1390]:
```suggestion
```

File: xml/System/Double.xml [hunk 5]
```diff
@@ -1380,18 +1380,24 @@
...
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Double Example/CPP/source.cpp" id="Snippet2":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" interactive="try-dotnet-method" id="Snippet2":::
```
Comment (mairaw) [inline xml/System/Double.xml:1392]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" id="Snippet2":::
```

File: xml/System/Double.xml [hunk 6]
```diff
@@ -1380,18 +1380,24 @@
...
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Double Example/VB/source.vb" id="Snippet2"::: 
+
+=======
```
Comment (mairaw) [inline xml/System/Double.xml:1395]:
```suggestion
```

File: xml/System/Double.xml [hunk 7]
```diff
@@ -1380,18 +1380,24 @@
...
  [!code-cpp[Double Example#2](~/samples/snippets/cpp/VS_Snippets_CLR/Double Example/CPP/source.cpp#2)]
```
Comment (mairaw) [inline xml/System/Double.xml:1396]:
```suggestion
```

File: xml/System/Double.xml [hunk 8]
```diff
@@ -1380,18 +1380,24 @@
...
- [!code-csharp[Double Example#2](~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs#2)]
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" id="Snippet2":::
```
Comment (mairaw) [inline xml/System/Double.xml:1397]:
```suggestion
```

File: xml/System/Double.xml [hunk 9]
```diff
@@ -1380,18 +1380,24 @@
...
  [!code-vb[Double Example#2](~/samples/snippets/visualbasic/VS_Snippets_CLR/Double Example/VB/source.vb#2)]
```
Comment (mairaw) [inline xml/System/Double.xml:1398]:
```suggestion
```

File: xml/System/Double.xml [hunk 10]
```diff
@@ -1380,18 +1380,24 @@
...
```
Comment (mairaw) [inline xml/System/Double.xml:1399]:
```suggestion
```

File: xml/System/Double.xml [hunk 11]
```diff
@@ -1380,18 +1380,24 @@
...
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" interactive="try-dotnet-method" id="Snippet2":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Double Example/VB/source.vb" id="Snippet2"::: 
+
+=======
  [!code-cpp[Double Example#2](~/samples/snippets/cpp/VS_Snippets_CLR/Double Example/CPP/source.cpp#2)]
- [!code-csharp[Double Example#2](~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs#2)]
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Double Example/CS/source.cs" id="Snippet2":::
  [!code-vb[Double Example#2](~/samples/snippets/visualbasic/VS_Snippets_CLR/Double Example/VB/source.vb#2)]  
   
+>>>>>>> 26c89fb445000818dccc0062b2bedac6ce92b085
```
Comment (mairaw) [inline xml/System/Double.xml:1400]:
```suggestion
```

File: xml/System/Double.xml [hunk 12]
```diff
@@ -1928,8 +1934,8 @@
   
  Ordinarily, if you pass the <xref:System.Double.Parse%2A?displayProperty=nameWithType> method a string that is created by calling the <xref:System.Double.ToString%2A?displayProperty=nameWithType> method, the original <xref:System.Double> value is returned. However, on .NET Framework and on .NET Core 2.2 and earlier versions, the values may not be equal because of a loss of precision. In addition, attempting to parse the string representation of either <xref:System.Double.MinValue?displayProperty=nameWithType> or <xref:System.Double.MaxValue?displayProperty=nameWithType> fails to round-trip. On .NET Framework and .NET Core 2.2 and previous versions, it throws an <xref:System.OverflowException>. The following example provides an illustration. 
   
- [!code-csharp[System.Double.Parse#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.Parse/cs/parse2.cs#3)]
- [!code-vb[System.Double.Parse#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Double.Parse/vb/parse2.vb#3)]  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.Parse/cs/parse2.cs" interactive="try-dotnet-class" id="Snippet3":::
```
Comment (mairaw) [inline xml/System/Double.xml:1937]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.Parse/cs/parse2.cs" interactive="try-dotnet-method" id="Snippet3":::
```

File: xml/System/Double.xml [hunk 13]
```diff
@@ -2068,8 +2074,8 @@ If a separator is encountered in the `s` parameter during a parse operation, and
 ## Examples  
  The following example uses the <xref:System.Double.Parse%28System.String%2CSystem.Globalization.NumberStyles%29> method to parse the string representations of <xref:System.Double> values using the en-US culture.  
   
- [!code-csharp[System.Double.Parse#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.Parse/cs/Parse.cs#1)]
- [!code-vb[System.Double.Parse#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Double.Parse/vb/Parse.vb#1)]  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.Parse/cs/Parse.cs" interactive="try-dotnet-class" id="Snippet1":::
```
Comment (mairaw) [inline xml/System/Double.xml:2077]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.Parse/cs/Parse.cs" id="Snippet1":::
```

File: xml/System/Double.xml [hunk 14]
```diff
@@ -3931,8 +3930,8 @@ This member is an explicit interface member implementation. It can be used only
...
   
- [!code-csharp[System.Double.TryParse#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.TryParse/cs/tryparse2.cs#3)]
- [!code-vb[System.Double.TryParse#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Double.TryParse/vb/tryparse2.vb#3)]  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.TryParse/cs/tryparse2.cs" interactive="try-dotnet" id="Snippet3":::
```
Comment (mairaw) [inline xml/System/Double.xml:3933]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Double.TryParse/cs/tryparse2.cs" interactive="try-dotnet-method" id="Snippet3":::
```

**Pull request merged**